### ISSUE TITLE
error message during boot about dnsmasq.host

### ISSUE THREAD

**Issue by LaurentBonnaud** [canonical/multipass#1118]
I am using multipass 0.8.1 under Ubuntu 19.10.
In the journal I see the following error message:

`Oct 08 13:30:20 vougeot dnsmasq[2032]: cannot read /var/snap/multipass/common/data/multipassd/network/dnsmasq.hosts: No such file or directory`

**Comment by brettkoonce** [comment]
i did a
    touch /var/snap/multipass/common/data/multipassd/network/dnsmasq.hosts
to make this disappear fwiw

**Comment by YousefSaber** [comment]
I face the same weird message 
```
joseph@joseph-X541UJ:~$ sudo systemctl status snap.multipass.multipassd.service 
● snap.multipass.multipassd.service - Service for snap application multipass.multipassd
   Loaded: loaded (/etc/systemd/system/snap.multipass.multipassd.service; enabled; vendor preset: enabled)
   Active: active (running) since Fri 2020-02-14 11:02:37 EET; 3min 15s ago
 Main PID: 7306 (multipassd)
    Tasks: 11 (limit: 4915)
   Memory: 34.0M
   CGroup: /system.slice/snap.multipass.multipassd.service
           ├─7306 /snap/multipass/1597/bin/multipassd --verbosity debug --logger platform
           └─7394 /snap/multipass/1597/usr/sbin/dnsmasq --strict-order --bind-interfaces --pid-file=/var/snap/multipass/common/data/multipassd

Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: compile time options: IPv6 GNU-getopt DBus i18n IDN DHCP DHCPv6 no-Lua TFTP conntrack ipset auth 
Feb 14 11:02:37 joseph-X541UJ dnsmasq-dhcp[7394]: DHCP, IP range 10.226.208.2 -- 10.226.208.254, lease time infinite
Feb 14 11:02:37 joseph-X541UJ dnsmasq-dhcp[7394]: DHCP, sockets bound exclusively to interface mpqemubr0
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: using local addresses only for domain multipass
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: reading /etc/resolv.conf
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: using local addresses only for domain multipass
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: using nameserver 127.0.0.53#53
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: read /etc/hosts - 7 addresses
Feb 14 11:02:37 joseph-X541UJ dnsmasq[7394]: cannot read /var/snap/multipass/common/data/multipassd/network/dnsmasq.hosts: No such file or directory
Feb 14 11:02:37 joseph-X541UJ multipassd[7306]: gRPC listening on unix:/var/snap/multipass/common/multipass_socket, SSL:on
```

**Comment by surahman** [comment]
I am trying to make sure I have correctly located the point at which the `dnsmasq` process is bootstrapped for the first time:
https://github.com/canonical/multipass/blob/981cf157b7c6348b2d55087b2630a26fc4cba531/src/platform/backends/qemu/dnsmasq_server.cpp#L46-L57

And would you want to purge what I am assuming is a DNS cache in `dnsmasq.hosts` on server shutdown? Essentially in the destructor.

**Comment by townsend2010** [comment]
This was fixed in #2099.

**Comment by LaurentBonnaud** [comment]
Thanks for the fix!
